src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -264,14 +273,34 @@ const ComposeForm = props => {
 
   useEffect(
     () => {
-      if (
-        messageInvalid ||
-        (isSignatureRequired && (signatureInvalid || checkboxInvalid))
-      ) {
-        focusOnErrorField();
-      }
+      const handleEffect = async () => {
+        const selectedTG = recipientsList.find(
+          recipient => recipient.triageTeamId === +selectedRecipientId,
+        );
+        if (
+          alertDisplayed &&
+          !alertHasFocused &&
+          alertRef.current &&
+          selectedTG
+        ) {
+          await focusElement(selectedTG?.name);
+          focusElement(alertRef.current);
```

Comment:
Can you help me understand lines 286 and 287?  What is the purpose of focusing on an element then immediately focusing on something else?

Reply:
@KyleCardwell This was done so that screen readers announced the selected recipient first before announcing the alert. Normally, when an alert is displayed it gets read immediately. However in this case, we still need to guarantee that the user is aware of their recipient selection before announcing the alert.

Reply:
wouldn't `aria-describedby` be a better implementation of this sequence? @BobbyBaileyRB